examples/raw-templates/raw_templates.description

```diff
@@ -6,3 +6,7 @@
 // </ul>
 //<br></br>
 // <p>Important use case: SQL parameters.</p>
+// Note that the relevant database driver JAR should be defined in the `Ballerina.toml` file as a dependency.
+// This sample is based on an H2 database and the H2 database driver JAR need to be added to `Ballerina.toml` file.
+// For a sample configuration and more information on the underlying module, see the [JDBC module](https://docs.central.ballerina.io/ballerinax/java.jdbc/latest/) .<br><br>
+// This sample is written using H2 2.1.210 and it is recommended to use H2 JAR with versions higher than 2.1.210.
```

Comment:
```suggestion
// For a sample configuration and more information on the underlying module, see the [JDBC module](https://docs.central.ballerina.io/ballerinax/java.jdbc/latest/) .<br><br>
// This sample is written using H2 2.1.210 and it is recommended to use H2 JAR with versions higher than 2.1.210.
```

Is this valid, since technically this doesnt use JDBC. Shall we give the link to the bbe ballerina.toml file?

Reply:
Changed here https://github.com/ballerina-platform/ballerina-distribution/pull/2558/commits/2cf351657e273bf1bf5427ff4c883c5ca1d24ba1

Reply:
@niveathika This BBE uses JDBC client. I assumed you meant this BBE not related to BBE but raw-template functionality.

Reply:
Created an issue to remove dependency https://github.com/ballerina-platform/ballerina-distribution/issues/2569